rootfs/etc/profile.d/banner.sh

```
@@ -28,6 +21,15 @@ if [ "${SHLVL}" == "1" ]; then
 	}
 
 	function _header() {
+		local ESC=$'\e'
+		local CYAN="${ESC}[36m"
+		local COLOR_RESET # Have to be careful because of dark mode
```

Shellcheck (0.10.0)

[warning] COLOR_RESET appears unused. Verify use (or export if used externally).

(SC2034)